lib/bundler/cli/inject.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
+require "bundler/cli/common"
```

Comment:
Ah, I was previously using `select_spec` from `CLI::Common`. Will remove this line as this isn't needed anymore.